src/__tests__/medications/search/ViewMedications.test.tsx

```diff
@@ -92,39 +79,46 @@ describe('View Medications', () => {
 
   describe('table', () => {
     it('should render a table with data with the default search', async () => {
-      const { wrapper } = await setup({} as Medication, [Permissions.ViewMedications])
-
-      const table = wrapper.find(MedicationRequestTable)
-      expect(table).toHaveLength(1)
-      expect(table.prop('searchRequest')).toEqual({ text: '', status: 'all' })
+      const { container } = await setup({} as Medication, [Permissions.ViewMedications])
+      await waitFor(() => {
+        expect(container.querySelector('table')).toBeInTheDocument()
+      })
+      expect(screen.getByLabelText(/medications\.search/i)).toHaveDisplayValue('')
     })
   })
 
   describe('search', () => {
     it('should render a medication request search component', async () => {
-      const { wrapper } = await setup({} as Medication)
+      setup({} as Medication)
 
-      const search = wrapper.find(MedicationRequestSearch)
-      expect(search).toHaveLength(1)
-      expect(search.prop('searchRequest')).toEqual({ text: '', status: 'all' })
+      const search = screen.getByLabelText(/medications\.search/i)
+      expect(search).toBeInTheDocument()
+      expect(search).toHaveDisplayValue('')
     })
 
     it('should update the table when the search changes', async () => {
       const expectedSearchRequest: MedicationSearchRequest = {
         text: 'someNewText',
         status: 'draft',
       }
-      const { wrapper } = await setup({} as Medication)
-
-      await act(async () => {
-        const search = wrapper.find(MedicationRequestSearch)
-        const onChange = search.prop('onChange')
-        await onChange(expectedSearchRequest)
+      const expectedMedicationRequests: Medication[] = [
+        {
+          id: 'someId',
+          medication: expectedSearchRequest.text,
+          status: expectedSearchRequest.status,
+        } as Medication,
+      ]
+      const { container } = await setup(
+        { medication: expectedSearchRequest.text } as Medication,
+        [],
+        expectedMedicationRequests,
+      )
+      userEvent.type(screen.getByLabelText(/medications\.search/i), expectedSearchRequest.text)
+      await waitFor(() => {
+        expect(container.querySelector('table')).toBeInTheDocument()
       })
-      wrapper.update()
 
-      const table = wrapper.find(MedicationRequestTable)
-      expect(table.prop('searchRequest')).toEqual(expectedSearchRequest)
+      expect(screen.getByText(expectedSearchRequest.text)).toBeInTheDocument()
```

Comment:
Looks like we lost an `expect` that verified the search was being performed correctly